search-commons/src/test/java/no/unit/nva/search/ticket/TicketClientTestV2.java

```diff
@@ -126,6 +131,29 @@ void shouldNotReturnFilesApprovalThesisWhenUserHasNoAccessRightToManageDegrees()
     assertTrue(response.hits().isEmpty());
   }
 
+  @ParameterizedTest
+  @EnumSource(
+      value = TicketType.class,
+      mode = Mode.EXCLUDE,
+      names = {"UNPUBLISH_REQUEST", "NONE"})
```

Comment:
UNPUBLISH_REQUEST is always filtered out. None is not a ticket type, not sure why it is included ticket types.